build_info/whois.control

```diff
@@ -0,0 +1,12 @@
+Package: whois
+Maintainer: @DEB_MAINTAINER@
+Architecture: @DEB_ARCH@
+Version: @DEB_WHOIS_V@
+Depends: libidn2, libcrypt1
```

Comment:
As far as I remember libcrypt1 isn't a real dependency ;)

Reply:
compiling mkpasswd as well to be inline with debian

libcrypt1 is needed for that.

Reply:
Is mkpasswd useful on a Darwin os? Have you tested it performs its intended function?

Reply:
I dunno if it's useful for anyone but I just included it in case someone expect it to be available in the whois package just like debian.

Reply:
I didn’t mean useful in the sense someone would have an interest in using it— I was more asking if it was even functional.

Reply:
Oh oops I forgot to mention that, yes it is.